### PR TITLE
:art: feat(Application design): Layout of the Application ref: #24

### LINKED ISSUE #24: 🎨  [Application design]  Layout of the Application
## Application layout

When a user logs in, they are navigated to the application homepage (Dashboard). Here they can view an analysis of all their bots and all the bots they are building and go to their builder.

---

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user I wish to view a navigation pane to other features of the application and a toolbar with my user menu.

### Description

The application layout at the moment looks like below:

Screenshot 2022-08-03 at 15 58 30

From here I can :
1. Interact with the sidenav at the left which contains my navigation.
2. Interact with the user component which contains my user menu.
3. Interact with the application pages which contain the various features of the application.

We have prepared a new design to improve the current UI as shown below:

Screenshot 2022-08-03 at 16 06 09

Your assignment, make it look like that picture. Ignore the sections filled in gray colour for now as those are in another feature.

### Technical Analysis

This original page layout is located in libs/elements/layout/page-convl/src/lib/components. The functionality is spread across the different components in this library.

For this feature you'll have to:
```markdown

1. Edit the convl-sidemenu component to match the new design.
2. Edit the convl-navbar component to match the new design.
```

### Acceptance Criteria

1. When rendered the component is pixel perfect compared to the image.
2. The component is well structured following the Technical Analysis

### PR DESCRIPTION
**Pull request by meshack34**
# Description
Reference the issue related to this PR as shown [here](https://github.com/italanta/italanta-apps/issues/24).
### Summary
*Fixes #24* completely
When a user logs in, they are able to navigated to the application homepage (Dashboard). Here they can view an analysis of all their bots and all the bots they are building and go to their builder.

**From here one can :**

Interact with the sidenav at the left which contains my navigation.
Interact with the user component which contains my user menu.
Interact with the application pages which contain the various features of the application.
We have prepared a new design to improve the current UI as shown below:

#
elewscreen

Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code

### CODE REVIEW

File: libs/elements/layout/page-convl/src/lib/components/convl-sidemenu/convl-sidemenu.component.scss [hunk 1]
```diff
@@ -2,35 +2,64 @@
...
+  max-width:100px;
+  height:40px;
   color: mat-color($accent, A200);
-  max-width: 25px;
-  border-radius: 5px;
+  padding-top:35px;
+}
+.category-children{
+  background-color:#E0C4EC;
+  border-right: 7px solid #351393;
```
Comment (ReaganCn) [inline libs/elements/layout/page-convl/src/lib/components/convl-sidemenu/convl-sidemenu.component.scss:60]: Use variables from _variables.scss for colors.

File: libs/elements/layout/page-convl/src/lib/components/convl-sidemenu/convl-sidemenu.component.scss [hunk 2]
```diff
@@ -110,5 +141,21 @@ mat-nav-list {
...
   cursor: pointer;
-  background-color: mat-color($accent, A200);
+  //background-color: mat-color($accent, A200);
+ 
+
+
+}
+
+.fa-gauge-high{
+  color:#351393;
```
Comment (ReaganCn) [inline libs/elements/layout/page-convl/src/lib/components/convl-sidemenu/convl-sidemenu.component.scss:151]: Same here. Use scss variables

**Comment by ReaganCn** [comment]
Hello Meshack, 
A few things to note:
- The logo is skewed. It should keep the original aspect ratio as shown below. This also inludes other elements in the side-nav
image

- If you are making the menu retractable, make sure that the rest of the page reacts the same(expands and contracts)

**Review by ReaganCn — CHANGES_REQUESTED** [review]

**Comment by meshack34** [comment]
Hi, I have made changes. Have a look

**Comment by ReaganCn** [comment]
You have just created the variable within the scss file and assigned it. We have a dedicated _variables.scss file that has the theme colors and you should use the variables from that file. It is located in libs/elements/theming/_variables.scss
Example:
`color: var(-- --convs-mgr-color-primary-yellow)`

**Comment by ReaganCn** [comment]
Issue resolved by Faiso on PR #99

**Pull request closed without merging**